src/main/steps/c100-rebuild/miam/miam-info/content.ts

```diff
@@ -0,0 +1,122 @@
+  line3Link:
+    '<a href="https://www.familymediationcouncil.org.uk/find-local-mediator/" class="govuk-link" target="_blank" aria-label="Find a mediator to book a MIAM">Find a mediator to book a MIAM</a>',
+  line4:
+    'You should have a document signed by the mediator confirming this. You need to upload this document to your application.',
+  line5:
+    'You do not have to attend a MIAM if you have a valid reason. For example, you or the children are at risk of harm.',
+  line6Link:
+    '<a href="https://www.gov.uk/check-legal-aid" class="govuk-link" target="_blank" aria-label="List of valid reasons for not attending a MIAM">List of valid reasons for not attending a MIAM</a>',
+  miamLabel: 'Attending a MIAM',
+  miamConsentStatement: 'I understand that I have to attend a MIAM or provide a valid reason for not attending.',
+  errors: {
+    miamConsent: {
+      required:
+        'Confirm that you understand that you have to attend a MIAM or provide a valid reason for not attending.',
+    },
+  },
+});
+
+const cy = () => ({
+  serviceName: 'Child Arrangements - welsh',
+  headingTitle: 'Attending a Mediation Information and Assessment Meeting MIAM - welsh',
+  paragraph1:
+    'Before completing this application you’re legally required to attend a Mediation Information and Assessment Meeting (MIAM), unless <a href="https://apply-to-court-about-child-arrangements.service.justice.gov.uk/about/miam_exemptions" class="govuk-link" target="_blank" aria-label=" you’re exempt."> you’re exempt.</a> - welsh',
+  paragraph2:
+    'A <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> is a meeting where you\'ll be given information about <a href="https://helpwithchildarrangements.service.justice.gov.uk/professional-mediation" class="govuk-link" target="_blank" aria-label="mediation,"> mediation,</a> and other ways to reach an agreement without going to court. A mediator will discuss with you whether these options are suitable for your case. - welsh',
+  legalAidLink:
+    '<a href="https://www.gov.uk/check-legal-aid" class="govuk-link" target="_blank" aria-label="Check if you’re eligible for legal aid during mediation">Check if you’re eligible for legal aid during mediation</a> - welsh',
+  miamMeeting:
+    'A <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> is a one-off meeting and not the same as mediation. - welsh',
+  miamHeading1: 'What happens at a <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> - welsh',
+  miamHeading2:
+    'If you’ve already attended a <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> - welsh',
+  miamHeading3:
+    'When you don’t have to attend a <abbr title="Mediation Information and Assessment Meeting">MIAM</abbr> - welsh',
+  miamList1: 'how mediation works - welsh',
```

Comment:
@rahulsanjava19 We can define this as an Array & use a for loop in template.njk to render.
Please refer starting guidance page or safety concerns guidance.